app/jsx/dashboard_card/DashboardColorPicker.js

```diff
@@ -121,14 +121,14 @@ import cx from 'classnames'
 
     leftPosition: function(){
       return this.tooltipOnRight() ?
-        (this.leftPlusElement() - 80) :
+        (this.leftPlusElement()) :
         (this.leftPlusElement() - 360)
     },
 
     pickerToolTipStyle: function() {
       if (this.props.isOpen) {
         return {
-          position: 'absolute',
+          position: 'fixed',
```

Comment:
@captainhurst I am considering leaving this here also for the time being. Now that we understand canvas better I am wondering if we can better shield out ongoing changes.